modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/preloader/GridDhtPartitionDemander.java

```diff
@@ -1213,6 +1227,8 @@ private void updateGroupMetrics() {
 
                 remaining.put(k.id(), v.partitions());
 
+                grp.metrics().addRebalancingPartitions(v.partitions().size());
```

Comment:
Let's rename to `rebalancingPartitionsLeft(value)`.
I think this should be set when the rebalance actually starts.